### DISCUSSION TITLE
Combining types

### DISCUSSION THREAD

**Discussion by YMHJERUSALEM** [jackvonhouse/custom-node-size#3] [Q&A] [answered]
Hi
is it possible to use your plugin, and still hold for some notes the regular Obsidian method?
Thanks

**Comment by jackvonhouse** [comment, marked as answer]
> Hi is it possible to use your plugin, and still hold for some notes the regular Obsidian method? Thanks

Of course! My plugin only modify notes that have `node_size` property, so your notes will not change in any way on the graph.

**Reply by YMHJERUSALEM** [reply]
Thank U!
